packages/enhanced_gradients/CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+## 0.0.1
+
+* TODO: Describe initial release.
```

Comment:
todo here

Reply:
I'm a bit confused as to how to properly write an initial changelog :/ Also, do we use or want to use `melos version`?